include/votca/csg/bead.h

```diff
@@ -15,49 +15,56 @@
  *
  */
 
-#ifndef _VOTCA_CSG_BEAD_H
-#define _VOTCA_CSG_BEAD_H
+#ifndef VOTCA_CSG_BEAD_H
+#define VOTCA_CSG_BEAD_H
 
-#include <assert.h>
 #include <cassert>
+#include <sstream>
 #include <string>
 #include <votca/tools/property.h>
 #include <votca/tools/types.h>
-#include <votca/tools/vec.h>
 
 #include "basebead.h"
 
 namespace votca {
 namespace csg {
 
-class Topology;
 class Molecule;
 
-using namespace votca::tools;
-
 /**
  * \brief information about a bead
  *
  * The Bead class describes an atom or a coarse grained bead. It stores
- * information like the id, the name, the mass, the
- * charge and the residue it belongs to. The coordinates are stored in the
- * configuration class.
+ * information like the id, the name, the mass, the charge and the residue it
+ * belongs to.
  *
- * \todo change resnr to pointer
- * \todo make sure bead belongs to topology
  **/
 class Bead : public BaseBead {
  public:
+  Bead(){};
   /**
    * destructor
    */
   virtual ~Bead() {}
 
+  int getResnr() const {
```

Comment:
deprecated features are not removed features. If you want to deprecate a feature put the 

https://en.cppreference.com/w/cpp/language/attributes

attribute then